Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -611,34 +619,34 @@ Describe "Test -EnableExit Switch" {
     }
 
     # using statements are only supported in v5+
-    if (!$testingLibraryUsage -and ($PSVersionTable.PSVersion -ge [Version]'5.0.0')) {
-        Describe "Handles parse errors due to unknown types" {
+    Describe "Handles parse errors due to unknown types" -Skip:($testingLibraryUsage -or ($PSVersionTable.PSVersion -lt [Version]'5.0.0')) {
```

Comment:
The `-lt` here should coerce the string on the RHS to be a version without the type

Reply:
Fixed in 6a44a18e3837119f61d7848232ef6897c094f6ed, lets hope it also works for WMF4